pkg/docstring/docstring.go

```diff
@@ -388,7 +388,10 @@ func parseBlockTitle(l string) (title string, ok bool) {
 }
 
 // fieldRe matches "field: ...".
-var fieldRe = regexp.MustCompile(`^(\S*)\s*:\s*(.*)$`)
+//
+// fix: Modify `^(\S*)\s*:\s*(.*)$` to `^(\S.*?)\s*:\s*(.*)$` will allow spaces,
+// and accept lines like `os (str): name of os`
```

Comment:
Can you change this to simply document how the regex works? Something like
```suggestion
// fieldRe matches arguments in docstrings, allowing spaces around the name and type
```

Reply:
Fine. Thank you for review.